server/src/modules/stats/board/board.stats.service.ts

```diff
@@ -0,0 +1,157 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model, Types } from 'mongoose';
+import { BoardStats } from 'src/schemas/stats/board.stats.schema';
+
+@Injectable()
+export class BoardStatsService {
+  constructor(
+    @InjectModel(BoardStats.name)
+    private readonly boardStatsModel: Model<BoardStats>,
+  ) {}
+
+  async initStats(boardId: string): Promise<void> {
+    await this.boardStatsModel.create({ boardId: boardId });
+  }
+
+  async removeStats(boardId: string): Promise<void> {
+    await this.boardStatsModel.findOneAndDelete({ boardId: boardId });
+  }
+
+  async updateLastAccessed(boardId: string): Promise<void> {
+    await this.boardStatsModel.updateOne(
+      { boardId: new Types.ObjectId(boardId) },
```

Comment:
I think there is no need for `new Types.ObjectId(boardId)` and it could be shortened to 
`await this.boardStatsModel.updateOne({ boardId })`

Reply:
same situation in other places - its not required - its just my suggestion